doc/manuals/simulation-configuration-file.md

```diff
@@ -299,6 +299,14 @@ The simulation configuration file accepts the following JSON properties or entri
+    * **external**: Information about an external source from which to load, to transform and to register  data into a Context Broker instance. The `external` property includes the following sub-properties:
+        * **body**: Optional. The body to be included into the requests sent to the external source to get the data.
+        * **collector**: Mandatory. The `collector` property includes Javascript code used to transform the data loaded from an external source into an array of arrays, each of them including Javascript objects corresponding to the attributes to be updated or notified. This is, in case only one entity including a textual attribute called `attr-001` wants to be updated or notified, the `collector` code should return an array of arrays such as `[[{name: 'attr-001', type: "Text", value: "Some text value"}]]`. If more than 1 array of attributes are returned, a sequence of requests will be sent, each one of them including the attributes returned in each one of the returned arrays. The `collector` may `require()` NPM packages (previously declaring them in the `require` property) just as in the `attribute-function-interpolator` interpolator and should return its result (i.e., the array of arrays) assigning it to the `module.exports` variable. An `external` variable is injected into the `collector` property Javascript code including the simulation configuration `external` property contents of the concrete entity augmented with an additional `response` sub-property with the result of the request made to the external endpoint. A valid value for the `collector` property is the following: `"var csvParse = require('csv-parse/lib/sync'); var data = external.response.body; var csvDataIndex = data.indexOf('\"Fecha y hora oficial'); var csvData = data.substring(csvDataIndex); var parsedCSVData = csvParse(csvData, {columns: true}); function getDate(dateString) { return new Date(dateString.substring(6,10), dateString.substring(3,5), dateString.substring(0,2), dateString.substring(11,13), dateString.substring(14));} var dateObserved = getDate(parsedCSVData[0]['Fecha y hora oficial']); module.exports = [[{name: 'dateObserved', type: 'DateTime', value: dateObserved}, {name: 'dateModified', type: 'DateTime', value: dateObserved}, {name: 'temperatura', type: 'Number', value: parsedCSVData[0]['Temperatura (�C)']}, {name: 'vientoVelocidad', type: 'Number', value: parsedCSVData[0]['Velocidad del viento (km/h)']}]];"`.
```

Comment:
Instead of saying "A valid value for...", I would say "For instance...", or "An example is...". "A valid value for..." may be interpreted as the unique valid value.

There are several "A valid value for..." statements like this one in the PR.